examples/job/expansions/job-expand.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/python
```

Comment:
First of all you're mixing tabs and spaces in this file, which is highly undesirable in python world. Please use either.